basic-javascript/project-2.spec.js

```diff
@@ -1,3 +1,142 @@
 const funcs = require('./project-2');
 
-// whoops.. there is no test suite for this file. You'll simply just have to create one :/
+describe("getBiggest", () => {
+    it("should get the biggest number", () => {
+        expect(funcs.getBiggest(9, 5)).toBe(9);
+        expect(funcs.getBiggest(3, 3)).toBe(3);
+        expect(funcs.getBiggest(3, 8)).toBe(8);
+    });
+});
+
+describe("greeting", () => {
+    it("should get a proper greeting depending on the language", () => {
+        expect(funcs.greeting()).toBe("Hello!");
+        expect(funcs.greeting("German")).toBe("Guten Tag!");
+        expect(funcs.greeting("Spanish")).toBe("Hola!");
+    });
+});
+
+describe("isTenOrFive", () => {
+    it("should return true if the num is 10 or 5", () => {
+        expect(funcs.isTenOrFive(7)).toBeFalsy();
+        expect(funcs.isTenOrFive(5)).toBeTruthy();
+        expect(funcs.isTenOrFive(10)).toBeTruthy();
+    });
+});
+
+describe("isInRange", () => {
+    it("should check if the num is in range between 20 and 50", () => {
+        expect(funcs.isInRange(3)).toBeFalsy();
+        expect(funcs.isInRange(24)).toBeTruthy();
+    });
+});
+
+describe("isInteger", () => {
+    it("should check if the num is integer", () => {
+        expect(funcs.isInteger(91.3)).toBeFalsy();
+        expect(funcs.isInteger("apple")).toBeFalsy();
+        expect(funcs.isInteger(7)).toBeTruthy();
+    });
+});
+
+describe("fizzBuzz", () => {
+    it("should return fizzbuzz if the num is divisible by 5 and 3, fizz if by 3, buzz if by 5, and the num itself if no condition is met", () => {
+        expect(funcs.fizzBuzz(3)).toBe("fizz");
+        expect(funcs.fizzBuzz(5)).toBe("buzz");
+        expect(funcs.fizzBuzz(45)).toBe("fizzbuzz");
+        expect(funcs.fizzBuzz(101)).toBe(101);
+    });
```

Comment:
Let's consider adding more descriptions for each test cases.
For example:
    it('returns `fizzbuzz` if the input is divisible by 5 and 3', () => {
        expect(funcs.fizzBuzz(15)).toBe('fizzbuzz');
        expect(funcs.fizzBuzz(45)).toBe('fizzbuzz');
    })
    it('returns `fizz` if the input is divisible by 3', () => {
        expect(funcs.fizzBuzz(3)).toBe('fizz');
        expect(funcs.fizzBuzz(6)).toBe('fizz');
        expect(funcs.fizzBuzz(9)).toBe('fizz');
    })
    it('returns `buzz` if the input is divisible by 3', () => {
        expect(funcs.fizzBuzz(5)).toBe('buzz');
        expect(funcs.fizzBuzz(10)).toBe('buzz');
        expect(funcs.fizzBuzz(20)).toBe('buzz');
    })
This could help the developer to see if he/she is on the right path.